### ISSUE TITLE
Community Edition has no detach()?

### ISSUE THREAD

**Issue by allen3371** [jsplumb/community-edition#712]
https://jsplumbtoolkit.com/community/apidocs/classes/jsPlumb.html
i cant find the jsplumb.detach() function
but in document
https://jsplumbtoolkit.com/community/doc/removing.html
the doc have teh detach() function
